test-dsl/src/main/groovy/com/smartbear/readyapi4j/dsl/assertions/SoapRequestAssertionsDelegate.groovy

```diff
@@ -1,14 +1,19 @@
 package com.smartbear.readyapi4j.dsl.assertions
 
 import com.smartbear.readyapi4j.assertions.Assertions
-
+import com.smartbear.readyapi4j.assertions.NotSoapFaultAssertionBuilder
+import com.smartbear.readyapi4j.assertions.SoapFaultAssertionBuilder
 
 class SoapRequestAssertionsDelegate extends AbstractAssertionsDelegate {
-    void soapFault() {
-        assertionBuilders.add(Assertions.soapFault())
+    SoapFaultAssertionBuilder getSoapFault() {
+        SoapFaultAssertionBuilder soapFault = Assertions.soapFault()
```

Comment:
I'd inline this variable and return null instead, then add a comment clarifying that it's not a real getter. It's a bit ugly but less confusing.